examples/spark-pi/README.md

```diff
@@ -0,0 +1,33 @@
+#Spark-pi example
+The aim of this example is to show how to run a Spark application in cluster mode on MANTL.
+
+##Prerequisites 
+* Install Spark throug the mantl-api:
+```bash
+curl -k -i -L -X POST -d "{\"name\": \"spark\"}" https://admin:password@control-node/api/1/install
+```
+>**Note**: in the previous command, don't forget to substitute *password* and *control-node* with the actual password, 
+and the actual control node IP (or domain name) of your MANTL cluster.
+
+* Install the GlusterFS addon: http://docs.mantl.io/en/latest/components/glusterfs.html
+
+* SSH into one of your nodes, and create a configuration file `/mnt/container-volumes/spark-conf/spark-defaults.conf` that 

```

Comment:
@siddharthist if you have multiple copies of the conf file, and you want to change the configuration, then  you will have to modify all of them. I find it easier to have only one copy on GlusterFS.